config/domains.js

```diff
@@ -0,0 +1,9 @@
+module.exports = [
+    'https://app.contentful.com',
+    'https://app.eu.contentful.com',
```

Comment:
Default values of the Live Preview SDK itself (+ localhost), but it does not hurt to be explicitly listed here.